x/superfluid/keeper/superfluid_asset_test.go

```diff
@@ -43,11 +43,38 @@ func (s *KeeperTestSuite) TestSuperfluidAssetSetGetDeleteFlow() {
 }
 
 func (s *KeeperTestSuite) TestGetRiskAdjustedOsmoValue() {
-	s.SetupTest()
+	type riskFactorTest struct {
+		name            string
+		riskFactorDenom string
+		riskFactor      string
+		queryDenom      string
+		initialAmt      int64
+		expected        int64
+	}
+
+	tests := []riskFactorTest{
+		{"test risk factor not set ", "random", "", "btc", 100, 50},
+		{"test risk factor set high", "btc", "0.8", "btc", 100, 20},
+		{"test risk factor set low", "btc", "0.1", "btc", 100, 90},
+		{"test risk set diff query", "gamm/pool/1", "0.1", "btc", 100, 50},         // default
+		{"test risk set same query", "gamm/pool/1", "0.1", "gamm/pool/1", 100, 90}, // set
```

Comment:
Same requests with these, would be nice to have struct names in front of each input to make understanding each test case easier.

Reply:
doesn't your editor display the types inline? Mine looks like this:
<img width="1343" alt="Cursor_and_Notification_Center" src="https://github.com/osmosis-labs/osmosis/assets/37868/7db2b8a5-cd66-4bd5-9994-b7bfb74e32a0">